templates/repo/pulls/viewed_files_summary.tmpl

```diff
@@ -0,0 +1,4 @@
+<meter id="viewed-files-summary" value="{{.Diff.NumViewedFiles}}" max="{{.Diff.NumFiles}}"></meter>
+<label for="viewed-files-summary" id="viewed-files-summary-label" data-text-changed-template="{{.i18n.Tr "repo.pulls.viewed_files_label"}}">
+	{{.i18n.Tr "repo.pulls.viewed_files_label" .Diff.NumViewedFiles .Diff.NumFiles}}
+</label>
```

Comment:
This template is only used once? Then should it be put into the parent template directly?